pubspec.yaml

```diff
@@ -46,7 +46,12 @@ dependencies:
 
   # UI Widgets
   cupertino_icons: ^0.1.3
-  flare_flutter: ^2.0.6
+  flare_flutter: ^2.0.3
+  auto_route: ^0.6.7
+
+dependency_overrides:
```

Comment:
You can remove all these `dependency_overrides` they are not needed anymore

Reply:
Done